pkg/resources/management.cattle.io/v3/nodedriver/NodeDriver.md

```diff
@@ -0,0 +1,11 @@
+## Validation Checks
+
+Note: checks only run if a node driver is being disabled or deleted
+
+### RKE1 Nodes
+
+RKE1 Nodes are under the `nodes.management.cattle.io` CRD and watched via a controller+cache. When a request for disable/deletion of the driver comes in the validation will list all of the nodes and look through the `status.NodeTemplateSpec.Driver` field to see if any of the nodes are using the NodeDriver that is being requested to be disabled, if any are found the request will be denied.
+
+### RKE2 Nodes via DynamicDriver
+
+RKE2 Nodes can be created via their own CRD's (think `digitaloceanmachine.rke-machine.cattle.io` for the DigitalOcean NodeDriver) and thus need to be looked up with the dynamic client. If there are _any_ instances of the CRD for that NodeDriver the request will be denied.
```

Comment:
Apologies, we just added these docs requirements, so the expected format may need a better explanation. The goal of these docs is to let users know what the webhook is checking in their cluster. So we do not want implementation details only high-level explanations. We are trying to emulate this page that is provided by K8s https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#what-does-each-admission-controller-do
We also only need one section per Admitter. So this file would give an overview of how the validation stops users from disabling or deleting a driver that is in use.